tests/SideBySide.New/ConnectionPool.cs

```diff
@@ -148,9 +118,5 @@ public void ResetConnection(object connectionReset, int poolSize, long expected)
 				connection.Dispose();
 		}
 
-		public void Dispose()
-		{
-			MySqlConnection.ClearAllPools();
```

Comment:
Ah, good find!  This should be an easy fix then - let's open another issue for it after this is merged.